venvs/make.py

```diff
@@ -10,12 +10,31 @@
 
 from functools import partial
 
+import filesystems
 from filesystems import Path
 from packaging.requirements import Requirement
 import click
+import tomlkit
 
 from venvs import __version__
-from venvs.common import _FILESYSTEM, _LINK_DIR, _ROOT
+from venvs.common import _FILESYSTEM, _LINK_DIR, _ROOT, dump_config, load_config
+
+
+def add_virtualenv_config(filesystem, locator, installs, links):
```

Comment:
So this makes things a bit asymmetric -- now creating a venv will persist it, but `rmenv`'ing it won't remove it. Not sure how that makes me feel yet.

Reply:
Maybe a `remove` command would help?

Reply:
I think now that it's not the default I feel OK here (with there not being a remove).